onnxruntime/core/providers/cuda/cuda_execution_provider.h

```diff
@@ -94,12 +95,21 @@ class CUDAExecutionProvider : public IExecutionProvider {
 
   std::unique_ptr<profiling::EpProfiler> GetProfiler() override;
 
+#if defined(CUDA_VERSION) && CUDA_VERSION >= 11000
+  void CaptureBegin() override;
+  void CaptureEnd() override;
+  void Replay() override;
+#endif
+
  private:
   CUDAExecutionProviderInfo info_;
   cudaDeviceProp device_prop_;
   bool external_stream_ = false;
   cudaStream_t stream_ = nullptr;
 
+#if defined(CUDA_VERSION) && CUDA_VERSION >= 11000
+  CUDAGraph graph_ = NULL;
```

Comment:
Is CUDAGraph a pointer ?

Reply:
It is not a pointer. `NULL` is removed now.